contrib/autotest/profiles/rs485-modbus-dmx-onewire-compile-test

```diff
@@ -949,11 +947,17 @@ IPV4_SUPPORT=y
 # _2232HIO is not set
 # _4232h is not set
 # _89isp is not set
+# C232HM is not set
```

Comment:
Yes, avrdude 6.3. But the list of valid programmers is build at make menuconfig time, so this shouldn't matter.